src/include/ipc/dai.h

```diff
@@ -95,6 +97,7 @@ struct sof_ipc_dai_config {
 		struct sof_ipc_dai_alh_params alh;
 		struct sof_ipc_dai_esai_params esai;
 		struct sof_ipc_dai_sai_params sai;
+		struct sof_ipc_dai_afe_params afe;
```

Comment:
What does "AFE" stand for? It's everywhere but I didn't find it expanded anywhere.

"AFE" is also triggering a huge amount of `checkpatch --codespell` false positives; feel free to submit a `scripts/checkpatch.pl` workaround

Reply:
AFE is the abbreviation for Audio Front End, which is mediatek audio hardware and we also use it in linux kernel, e.g. https://elixir.bootlin.com/linux/latest/source/sound/soc/mediatek/mt8192/mt8192-afe-common.h

Reply:
> AFE is the abbreviation for Audio Front End

Thanks! Can you make sure it is expanded at least once in at least one top-level file? Kconfig, CMakeLists.txt, top-level .h or other top-level file. Sorry if I missed it.